### PR TITLE
Update RFC

### PR DESCRIPTION
**Pull request by anngvu**
Add contributors, bib, respondent summary, and minor prose and heading changes

### CODE REVIEW

**Comment by anngvu** [comment]
@allaway Need your help to add contributor names. We only have emails from the original survey results, and I don't know most of them.

File: rfc-brief.Rmd [hunk 1]
```diff
@@ -16,13 +16,18 @@ author:
   affiliation: Sage Bionetworks
   affiliation_url: https://sagebionetworks.org/
   orcid_id: 0000-0003-1488-6730
-date: "`r Sys.Date()`"
+- name: |
+    </br><span class = "custom-heading">Community Contributors</span></br>
+- name: Larry Benowitz, Joanne Ngeow, Riccardi ?, S Plotkin, Jian Qiang, Angela Hirbe, Filipa Jorge, Shruti Garg, R. Oostenbrink, B? Korf, Hui Liu, Adrienne ?, Parnal Joshi, Tom Reh
```
Comment (allaway) [inline rfc-brief.Rmd:21]:
```suggestion
- name: Larry Benowitz, Joanne Ngeow, Vincent Riccardi, Scott Plotkin, Jianqiang Wu, Angela Hirbe, Filipa Jorge Teixeira, Shruti Garg, Rianne Oostenbrink, Bruce Korf, Hui Liu, Adrienne Watson, Parnal Joshi, Tom Reh, Eva Trevisson
```

**Review by allaway — APPROVED** [review]
Looks good to me! I made some changes to the contributor list.

**Comment by allaway** [comment]
Oops, merged too fast - guess we also need to update the html.

**Pull request merged**